### PR TITLE
[WIP - SAF 282] Implement self reporting screens

### PR DESCRIPTION
**Pull request by zhaccc**
#### Description:

This PR fixes comments on the existing PR that was open a while ago but abandoned. It also implements new design and does some tweaks to existing code.

#### Screenshots:

self-assessment

### CODE REVIEW

File: app/views/assessment/survey.en.json [hunk 1]
```diff
@@ -0,0 +1,399 @@
+{
```
Comment (tparesi) [inline app/views/assessment/survey.en.json:1]: Will this be translated properly or does it have to be moved under the locale folder (specifically en.json)?

**Review by tparesi — APPROVED** [review]
Really hard to get a feel for all the changes, but I think it all looks good. I know this is a draft at this point, but I don't see much leftover.

File: app/locales/en.json [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+    "emergency_cta": "Call Emergency Services",
+    "isolate_title": "Isolate Yourself",
+    "isolate_description": "You <1>have some symptoms</1> that may be related to COVID-19. Call your healthcare provider if your symtoms get worse. Start home isolation.",
+    "isolate_cta": "Next",
+    "share_title": "Publish anonymized data",
+    "share_description": "During the interview with <HA name> your interviewer can remove locations you would like to keep private, such as homes and apartments. They can also add any additional locations you believe should be included.\n\n<HA name> may share the GPS coordinates from your location history, including date, time, and duration of your visits to let others in your community know if they may have crossed paths with the virus.\n\nI understand that the <HA name> representative will ask me verbally at the end of the interview if I consent.",
+    "share_screening_title": "Anonymized Screening Data",
+    "share_screening_description": "Share anoymized answers to screening questions with your local healthcare authority.",
```
Comment (mattThousand) [inline app/locales/en.json]: ~anoymized~ anonymized

**Review by johnschoeman — APPROVED** [review]
THIS IS A BIG BOY.

I think theres a lot of improvements that could be made, in terms of consistency and factoring, that said, for sake of getting this long running branch in, let's merge it in and make any future code changes incrementally.

**Pull request merged**